lxd/daemon.go

```diff
@@ -1664,10 +1664,10 @@ func (d *Daemon) Stop(ctx context.Context, sig os.Signal) error {
 		// Full shutdown requested.
 		if sig == unix.SIGPWR {
 			logger.Info("Stopping instances")
-			instancesShutdown(s, instances)
+			_ = instancesShutdown(s, instances)
```

Comment:
I think we can alter `instancesShutdown` to not return an error at all, as it actually never does :)